package.json

```diff
@@ -35,5 +35,22 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/232685646-e6cb718f-86f1-4f82-bbdc-51372b56edf9.png)

- [ ] As you can see in the image above that some requirements are missing 😨 First you should open a new issue in this repository and then include the following information in that issue 🤔 

* Github Project Link (Link to Kanban Board you have created)
* Information about the total number of members in this project

## Example ⬇️
![image](https://user-images.githubusercontent.com/93039359/232686130-312a0962-bdc8-43d5-a7c5-818bdfadc465.png)